pontoon/api/schema.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import absolute_import
+
```

Comment:
Shouldn't we remove this? https://github.com/mozilla/pontoon/commit/142f75013c691762e60ec1e69af0a9320cc14b45